src/index.html

```diff
@@ -9,8 +9,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form-big"
+    action="/"
+    method="get"
+    data-qa="big"
+    >
+      <label class="search">
+        <img
+        src="images/Search.svg"
+        alt="search"
+        class="img1"
+        >
+        <input
+        class="input-big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+
+      </label>
```

Comment:
blank line should be between same level elements, not in the end
```suggestion
      <label class="search">
        <img
          src="images/Search.svg"
          alt="search"
          class="img1"
        >
        
        <input
          class="input-big"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
```